test/integration/ci-operator-prowgen/output/jobs/private-org/super/private-org-super-master-presubmits.yaml

```diff
@@ -0,0 +1,57 @@
+presubmits:
```

Comment:
test for skip cloning?

Reply:
We should already have those tests. see [here](https://github.com/openshift/ci-tools/blob/master/test/integration/ci-operator-prowgen/output/jobs/private-org/duper/private-org-duper-master-presubmits.yaml#L1-L58)